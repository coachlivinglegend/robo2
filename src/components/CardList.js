import React from 'react';
import Card from './Card';
const CardList = ({ robots }) => {
  console.log('CardList');

  return (
    <div>
      {robots.map(({ name, email }, idx) => (
        <Card key={idx} name={name} email={email} id={idx} />
      ))}
    </div>
  );
};

export default CardList;
