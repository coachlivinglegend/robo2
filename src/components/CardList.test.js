import { shallow } from 'enzyme';
import CardList from './CardList';

test('expect to take a snapshot', () => {
  const robots = [
    {
      id: 1,
      name: 'John Doe',
      username: 'joneddidoe',
      email: 'sssskkkkkkkkkrrrrrrrrr@io',
    },
    {
      id: 2,
      name: 'John Doe',
      username: 'joneddidoe',
      email: 'sssskkkkkkkkkrrrrrrrrr@io',
    },
  ];
  const wrapper = shallow(<CardList robots={robots} />);
  expect(wrapper).toMatchSnapshot();
});
