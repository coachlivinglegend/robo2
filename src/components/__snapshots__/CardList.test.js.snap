// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to take a snapshot 1`] = `
<div>
  <Card
    email="sssskkkkkkkkkrrrrrrrrr@io"
    id={0}
    key="0"
    name="John Doe"
  />
  <Card
    email="sssskkkkkkkkkrrrrrrrrr@io"
    id={1}
    key="1"
    name="John Doe"
  />
</div>
`;
